package.json

```diff
@@ -36,7 +36,7 @@
     "zone.js": "^0.6.23"
   },
   "devDependencies": {
-    "@angular/cli": "^1.0.0",
+    "@angular/cli": "^1.0.2",
```

Comment:
@nikhilrayaprolu I prefer the first method too, it would be easier to follow and look up.